schedule/yast/autoyast/autoyast_multipath.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: autoyast_installonly
```

Comment:
probably 'autoyast_multipath' is what you meant.

Reply:
Nice catch, will fix this one. Thanks!